### PR TITLE
feat: add C# code snippets

### PR DESCRIPTION
**Pull request by TimBo93**
Examples for .NET are missing and the required
```
 ServerCompatibilityMode=NoTypeLoading;
```
might be a pitfalf.

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks Tim, added example looks good, we'll enable syntax highlighting for csharp, this can be done after merging though

**Comment by ideoma** [comment]
ServerCompatibilityMode=NoTypeLoading; is not necessary on latest version in master, incompatibility is fixed

File: docs/develop/connect.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ import TabItem from "@theme/TabItem"
   { label: "C", value: "c" },
   { label: "Python", value: "python" },
   { label: "psql", value: "psql" },
+  { label: ".NET", value: "csharp" },
```
Comment (mpsq) [inline docs/develop/connect.md:32]:
```suggestion
  { label: "C#", value: "csharp" },
```

**Review by mpsq — CHANGES_REQUESTED** [review]

**Review by mpsq — APPROVED** [review]

**Comment by mpsq** [comment]
@TimBo93 Thanks for the contribution!
@ideoma Would it be possible to test the snippet and confirm that it works before I merge it?

**Review by mpsq — CHANGES_REQUESTED** [review]
testing

**Review by mpsq — CHANGES_REQUESTED** [review]
This still needs some work before merging it: 

1. The snippet should be moved to develop/query-data
2. Some of the code in the snippet could be used to add an example to "develop/connect"
3. "C#" in the list of languages should be moved next to "Java"
4. The snippet should run the query "SELECT x FROM long_sequence(5);" like other snippets do, this is easier for the users since they can just copy/paste the code and run it without having to create a schema

**Review by mpsq — APPROVED** [review]

**Comment by mpsq** [comment]
Thanks to @TimBo93 and @bsmth :tada:

**Pull request merged**